src/mlpack/methods/random_forest/extra_trees_main.cpp

```diff
@@ -0,0 +1,259 @@
+
+*@file methods / random_forest / extra_trees_main.cpp *@author Pranshu Srivastava **A program to implement the extra trees algorithm ***mlpack is free software;
+you may redistribute it and / or modify it under the
+                                         *terms of the 3 -
+                                     clause BSD license.You should have received a copy of the * 3 - clause BSD license along with mlpack.If not,
+    see
+            *http : //www.opensource.org/licenses/BSD-3-Clause for more information.
+                    * /
```

Comment:
I'm not sure what happened with this comment... can you fix it please? :)

Reply:
yes, @rcurtin  I would need some help in writing tests as I have never written any before. Could you provide me with some resources where I can understand how to write tests?

Reply:
Hmm, you could probably start by imitating some of the tests in `src/mlpack/tests/main_tests/random_forest_main_test.cpp`. :+1:  Make sure to read through and understand the tests, and see if they make sense to adapt to this situation.

I'd also suggest adding the extra trees binding in `src/mlpack/methods/random_forest/CMakeLists.txt`, and then you should be able to test that it builds from the command-line with `make mlpack_extra_trees`. :+1:

Reply:
Yes, I have been trying to do exactly that. I will submit the PR in a few hours, Thanks

Reply:
Hey, @rcurtin  I was very unsure about the changes but please check them out.